### ISSUE TITLE
Update Readme

### ISSUE THREAD

**Issue by dewski** [NZKoz/cassandra_object#15]
The readme installation is a bit out of date. Could this be updated?

**Comment by lpgauth** [comment]
Yes, please. I'm trying to use this, but I have no clue how to setup the connection to Cassandra and there's no documentation...

**Comment by SethThomas** [comment]
I agree.  I'd love to see either a "Hello World" Rails3 app using cassandra_object or a quick tutorial on how to get started once the software is installed.
